src/pipeline_configuration_spec.py

```diff
@@ -45,11 +45,11 @@ class PipelineConfiguration:
 
     engagement_database: EngagementDatabaseClientConfiguration
     uuid_table: UUIDTableClientConfiguration
-    analysis_configs: AnalysisConfiguration
     project_start_date: datetime = None
     project_end_date: datetime = None
     test_participant_uuids: [] = None
     description: str = None
     rapid_pro_sources: [RapidProSource] = None
     coda_sync: CodaConfiguration = None
     rapid_pro_target: RapidProTarget = None
+    analysis_configs: AnalysisConfiguration = None
```

Comment:
(drive-by comment, perhaps for another PR - This only holds one AnalysisConfiguration, so should be analysis_config not analysis_configs)

Reply:
Thanks will update in a future PR.